firefox_puppeteer/ui/browser/window.py

```diff
@@ -8,12 +8,16 @@
 from marionette_driver.errors import NoSuchWindowException
 from marionette_driver.keys import Keys
 
-from firefox_puppeteer.ui.windows import BaseWindow
+from firefox_puppeteer.ui.windows import BaseWindow, Windows
 import firefox_puppeteer.errors as errors
 
 from firefox_puppeteer.api.l10n import L10n
 from firefox_puppeteer.decorators import use_class_as_property
 from firefox_puppeteer.api.prefs import Preferences
+from ..pageinfo.window import PageInfoWindow
+from .toolbars import NavBar
+from .tabbar import TabBar
+from ..about_window.window import AboutWindow
```

Comment:
Lets follow the absolute module path pattern like `from firefox_puppeteer.ui.pageinfo.window import PageInfoWindow`.